### ISSUE TITLE
Mode option not available with npm

### ISSUE THREAD

**Issue by cbinet-sidlee** [Dschungelabenteuer/custom-property-extract-loader#1]
Hi! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `custom-property-extract-loader@1.2.1` for the project I'm working on.

When using npm install to dowload the latest version of your package, we don't have the 'mode' option available. It looks like your version here on github is not up to date on npm. I had to manually update the options.json to make it work.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/custom-property-extract-loader/.DS_Store b/node_modules/custom-property-extract-loader/.DS_Store
new file mode 100644
index 0000000..3b05484
Binary files /dev/null and b/node_modules/custom-property-extract-loader/.DS_Store differ
diff --git a/node_modules/custom-property-extract-loader/dist/options.json b/node_modules/custom-property-extract-loader/dist/options.json
index 9c10ab7..5ae5dab 100644
--- a/node_modules/custom-property-extract-loader/dist/options.json
+++ b/node_modules/custom-property-extract-loader/dist/options.json
@@ -1,6 +1,11 @@
 {
   "type": "object",
   "properties": {
+    "mode": {
+      "description": "Output mode",
+      "type": "string",
+      "enum": ["full", "simple"]
+    },
     "syntax": {
       "description": "Syntax of the stylesheet",
       "type": "string",
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by Dschungelabenteuer** [comment]
Hey ! Thanks for reporting that issue ! I've just published  `custom-property-extract-loader@1.2.2` which addresses it. Please let me know if fixes your issue :)

**Comment by cbinet-sidlee** [comment]
Yes that fixed it ! Thanks a lot for the quick update 🙂
